app/src/main/java/com/codesoom/assignment/domain/Task.java

```diff
@@ -0,0 +1,43 @@
+package com.codesoom.assignment.domain;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * 할 일을 표현하는 클래스
+ * ID 와 TITLE 을 통해 표현한다.
+ */
+public class Task {
+    public static long taskSequence = 0;
+    private final Long id;
+    private String title;
+
+    private Task() {
+        this.id = ++taskSequence;
```

Comment:
여기도 다음과 같이 사용하는게 좋을 것 같아요.
```suggestion
        taskSequence++;
        this.id = taskSequence;
```

Reply:
넵 변경하였습니다!